skrobot/models/primitives.py

```diff
@@ -152,13 +170,30 @@ class MeshLink(model_module.Link):
 
     def __init__(self,
                  visual_mesh=None,
-                 pos=(0, 0, 0), rot=np.eye(3), name=None):
+                 pos=(0, 0, 0), rot=np.eye(3), name=None, with_sdf=False,
+                 dim_grid=None, padding_grid=None):
         if name is None:
             name = 'meshlink_{}'.format(str(uuid.uuid1()).replace('-', '_'))
 
         super(MeshLink, self).__init__(pos=pos, rot=rot, name=name)
         self.visual_mesh = visual_mesh
 
+        if with_sdf:
+            assert os.path.isfile(visual_mesh),\
+                "with_sdf is valid only with a mesh file"
+            if dim_grid is None:
```

Comment:
How about set dim_grid=100 instead of dim_grid=None.
padding_grid is same as.

Reply:
I agree.